tests/tests/optionsTests.robot

```diff
@@ -98,31 +91,33 @@ Enable Debug Output functionality test
     [Documentation]    Enable debug output functionality adds debug logs if selected
     [Tags]    options
 
+    ${test_file}=   Select random file
     Click Add Files Button
-    Upload Test File    ${file_4mb}[path]
+    Upload Test File    ${test_file}[path]
     Click Load Button
-    Check If Log Console Contains    Opening "${file_4mb}[name]"
-    Validate Output Description    ${file_4mb}[archive_name]
     Wait Until Page Does Not Contain Element    ${ExtractAndSaveDisabledButton}
-    Click Extract And Save Button    ${extraction_timeout}
-    Validate File Details In Log Console    ${file_4mb}
+    Check If Log Console Contains    Opening "${test_file}[name]"
+    Validate Output Description    ${test_file}[archive_name]
+    Click Extract And Save Button    ${test_file}[extraction_time]
+    Validate File Details In Log Console    ${test_file}
     Check If Log Console Does Not Contain    loaded
     Click Element    ${OptionsButton}
     Wait Until Element Is Visible    ${OptionsList}
     Click Element    ${EnableDebugSwitch}
+    Wait Until Element Is Visible    ${ReloadBadge}
     Click Load Button
-    Check If Log Console Contains    Opening "${file_4mb}[name]"
-    Validate Output Description    ${file_4mb}[archive_name]
     Wait Until Page Does Not Contain Element    ${ExtractAndSaveDisabledButton}
-    Click Extract And Save Button    ${extraction_timeout}
-    Validate File Details In Log Console    ${file_4mb}
+    Check If Log Console Contains    Opening "${test_file}[name]"
+    Validate Output Description    ${test_file}[archive_name]
+    Click Extract And Save Button    ${test_file}[extraction_time]
+    Validate File Details In Log Console    ${test_file}
     Check If Log Console Contains    loaded
-    Wait Until Element Is Visible    ${ReloadBadge}
-
+    
 Exclude temporary files functionality test
-   
-    [Documentation]    Exclude temporary files functionality removes tmp folder from loaded file
+    [Documentation]   Exclude temporary files functionality removes tmp folder from loaded file
     [Tags]    options
+
+    ${extraction_timeout}    Set Variable    60s
```

Comment:
You have "extraction timeout" as normal variable here, while in other tests it is taken from "file" variable